### PR TITLE
Added freenet.de preparation instructions

### LINKED ISSUE #139: freenet.de needs preparation
I setup a freenet.de account for a friend yesterday and it needed preparation.
One needs to login to the website, go to settings and activate the IMAP thing there.

freenet.de sends an E-Mail to inbox if one tries to login without doing that first.

### PR DESCRIPTION
**Pull request by missytake**
(I accidentally pushed to master, instead of a new branch. As I can't force-push on master, I decided to just revert the commit, so master is as before again. Now I reverted the revert commit, which means that this PR contains the changes I wanted to make in the first place. Always think twice before doing git commands, children.)

I didn't test this myself. @angelo-fuchs you tested this, right? Would these instructions have helped your friend?

closes #139

### CODE REVIEW

File: _providers/freenet.de.md [hunk 1]
```diff
@@ -14,4 +14,13 @@ server:
...
 website: https://www.freenet.de/index.html
+before_login_hint: To log in to freenet.de with Delta Chat, you need to activate POP3/IMAP/SMTP in the freenet.de web interface first.
 ---
+
+To use a freenet.de email address with Delta Chat, you need to activate
+POP3/IMAP/SMTP first in the web interface.
+
+You can find it at the "Postfach-Einstellungen", in the category
+"POP3/IMAP/SMTP" at "Einrichtung POP3/IMAP".  Here you need to activate the
+checkbock „Jetzt POP3, IMAP und SMTP aktivieren“ and click on "Speichern"
```
Comment (r10s) [inline _providers/freenet.de.md:25]: checkbox

**Review by r10s — COMMENTED** [review]
why not write the whole text in german? the provider has a german website only, and also the domain is .de only

File: _providers/freenet.de.md (hunk 1 above)
Comment (missytake) [reply _providers/freenet.de.md]: Should we have a policy about this?

Comment (missytake) [reply _providers/freenet.de.md]: (oh, and what does nb mean in this context? I only know it as "nonbinary"^^)

Comment (r10s) [reply _providers/freenet.de.md]: "nota bene", https://en.wikipedia.org/wiki/Nota_bene

Comment (r10s) [reply _providers/freenet.de.md]: > Should we have a policy about this?

hm, we could add a line to the README, a "policy" seems a bit too much :)

**Review by r10s — APPROVED** [review]

File: _providers/freenet.de.md [hunk 2]
```diff
@@ -14,4 +14,12 @@ server:
   type: smtp
 last_checked: 2017-06
 website: https://www.freenet.de/index.html
+before_login_hint: Um deine freenet.de E-Mail Adresse mit Delta Chat zu benutzen, musst du erst auf deren Webseite "POP3/IMAP/SMTP" aktivieren.
```
Comment (r10s) [inline _providers/freenet.de.md:17]: i think, "deren" is ambiguous here,

```suggestion
before_login_hint: Um deine freenet.de E-Mail-Adresse mit Delta Chat zu benutzen, musst du erst auf der freenet.de-Webseite "POP3/IMAP/SMTP" aktivieren.
```

File: _providers/freenet.de.md [hunk 3]
```diff
@@ -14,4 +14,12 @@ server:
...
 ---
+
+Um deine freenet.de E-Mail Adresse mit Delta Chat zu benutzen, musst du erst
+auf deren Webseite "POP3/IMAP/SMTP" aktivieren.
```
Comment (r10s) [inline _providers/freenet.de.md:21]:
```suggestion
Um deine freenet.de E-Mail-Adresse mit Delta Chat zu benutzen, musst du erst
auf der freenet.de-Webseite "POP3/IMAP/SMTP" aktivieren.
```

**Pull request merged**